src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
> _shrugs_ call me lazy but I personally prefer to follow PSR-1 & -2. Despite their flaws (spaces instead of tabs, funky braces, and what not I don't agree with) it is the standard we currently have. Honestly I just follow PSR-1 & -2 blindly in my own projects because it is easy and I don't have to think about it.

> We already have PSR 1 and 2, and I don't have good enough reason to not follow it.

That's exactly what I said. Most users just follow blindly whatever FIG proposes, even if it's flawed. Just for the sake of it being named `PSR`.

It's a code style, it's nothing that really needs one common standard. But well, I'm already happy that they didn't merge their own rules like the `Interface` suffix into PSR-2.